modules/find-policy-violations.adoc

```diff
@@ -0,0 +1,30 @@
+// Module included in the following assemblies:
+//
+// * operating/manage-cluster-configuration.adoc
+:_module-type: PROCEDURE
+[id="find-policy-violations_{context}"]
+= Finding policy violations
+
+[role="_abstract"]
+The *Policy Violations by Severity* widget in the *Configuration Management* view displays policy violations in a sunburst chart.
```

Comment:
In a future PR you could consider moving this conceptual content to a concept module that is included before this procedure module.